projects/minesweeper/src/main/java/org/batfish/minesweeper/bdd/ModelGeneration.java

```diff
@@ -0,0 +1,212 @@
+package org.batfish.minesweeper.bdd;
```

Comment:
This whole file is merely a refactoring of code that used to live in `SearchRoutePoliciesAnswerer`.